microshift_running_apps/microshift-operators-olm.adoc

```diff
@@ -0,0 +1,62 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="olm-with-microshift"]
+= Using Operator Lifecycle Manager with {microshift-short}
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-operators-olm
+
+toc::[]
+
+OLM is used in {microshift-short} for installing and running optional link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/architecture/control-plane#olm-operators_control-plane[add-on Operators].
+
+* Cluster Operators as applied in {ocp} are not used in {microshift-short}.
+* You can add your own catalogs for the Operators you want to use with your applications.
+* You must use the CLI to conduct OLM activities with {microshift-short}. The console and OperatorHub GUIs are not available, but you can take all the same actions with the CLI.
```

Comment:
@ShaunaDiaz for restricted network, it will create ISCP if you refer to ocp doc. but there is no ISCP in micorshift, 
Per reply in https://issues.redhat.com/browse/OPRUN-3063 by 
Doug Hellmann, he mentions `edge device users are expected to configure the system to use a mirror by updating cri-o's configuration directly`. it is important restriction for microshfit. it is better to mention ti.